dev-libs/hiredis/hiredis-1.0.0.ebuild

```diff
@@ -10,12 +10,14 @@ HOMEPAGE="https://github.com/redis/hiredis"
 SRC_URI="https://github.com/redis/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="BSD"
-SLOT="0/1.0.0"
+SLOT="0/$(ver_cut 1-2)"
```

Comment:
Please don't do this unless you're **absolutely** sure about upstream breaking ABI every major version. It necessarily doesn't happen.

Reply:
Ok, thanks, I'll remove it.